rse-project.Rmd

```diff
@@ -692,6 +692,53 @@ Breaking this down:
 > Documentation that serves these two groups well
 > will almost certainly serve the needs of strangers as well.
 
+These additional rules are taken from [@Good2014,@Mich2015,@Hart2016,@Zook2017]:
+
+-   **Keep raw data raw.**
+    Exactly what constitutes "raw" data isn't always clear:
+    the analog inputs to a voltmeter probably don't need to be recorded on tape
+    if we have the voltmeter's digitized output.
+    However,
+    our analysis methods and tools can change over time,
+    and even after years of work they can contain bugs.
+    We should therefore keep the original data exactly as we received it
+    so that we can re-do previous analyses or try new ones.
+    This is also a good safeguard if anyone ever questions the integrity of the research:
+    it will be hard to defend ourselves against accusations of fraud
+    if we can't show our work.
+
+-   **Give every part of the data a unique identifier.**
```

Comment:
"every part of the data" is extremely unclear here.  This reads like every cell in a dataset should have a DOI, which is safe to presume not the intent.  Can this be revised a bit?  Every datapoint in a dataset should have an "address" or location description that can uniquely identify it. For example, a data point may be within file ACX-02.csv, under column reading_type, and in row 450. 

I'm not sure I like to see discussion of persistent identifiers and row/column ids within the same discussion.  File names, column names, and entity IDs should be unique, meaningful, and their patterns/meaning documented.  Datasets should have DOIs once used for a publication.

Reply:
Yes that wording was unclear to me at first as well. I also read it as it referred to every cell or variable.

Reply:
changes look good